.travis.yml

```diff
@@ -1,14 +1,21 @@
 language: ruby
 
-cache: bundler
```

Comment:
could you try to set 'cache: false' only to 'ship' stage? same with bundler 2.0.1